### PR TITLE
Adds a new api_key field to province. 

### PR DESCRIPTION
**Pull request by PierrePaul**
This new field is used to make the call to COVID alert server to generate the one time key. If the province has no api_key, it will uses the default one provided by the system

https://trello.com/c/oyDDKKvN/276-add-a-token-field-to-the-provinces-table

Screenshot_2020-08-17 Modification de healthcare province COVID Health Portal administration Administration du Portail Aler

### CODE REVIEW

File: profiles/models.py [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 from django.utils.translation import gettext as _
 from django.contrib.auth.models import BaseUserManager, AbstractBaseUser
+from django.conf import settings
 from phonenumber_field.modelfields import PhoneNumberField
 
 
 class HealthcareProvince(models.Model):
     name = models.CharField(max_length=100, unique=True)
     abbr = models.SlugField(max_length=3, allow_unicode=True, unique=True)
+    api_key = models.CharField(null=True, blank=True, max_length=256)
```
Comment (pcraig3) [inline profiles/models.py:12]: Can we call it a "Bearer token" in the admin interface? (If it's not trivial to change the label, then let's leave it as is).

File: profiles/admin.py [hunk 2]
```diff
@@ -12,13 +12,14 @@
 
 class ProvinceAdmin(admin.ModelAdmin):
     list_display = ["id", "abbr", "name"]
```
Comment (pcraig3) [inline profiles/admin.py:14]: Is it possible to show the token column but with a "true/false" type value in the list view of the admin "list" interface? That way we could see at a glance which provinces we have set tokens for.

If not then let's double-check with Courtney if it's okay to show them on that page. I think it's probably better to.

File: profiles/models.py (hunk 1 above)
Comment (PierrePaul) [reply profiles/models.py:12]: It is trivial, will update.

File: profiles/admin.py (hunk 2 above)
Comment (PierrePaul) [reply profiles/admin.py:14]: It is trivial to do, will update! Thanks!

**Pull request merged**